packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -2738,27 +2738,36 @@ class Navigator extends StatefulWidget {
   /// the given context, and [Route.dispose] it.
   ///
   /// {@template flutter.widgets.navigator.removeRoute}
-  /// The removed route is removed without being completed, so this method does
-  /// not take a return value argument. No animations are run as a result of
-  /// this method call.
+  /// No animations are run as a result of this method call.
```

Comment:
It looks like this section was written 7 years ago; in my opinion it's fine to change the behavior if it fits a current use case 👍